grakn-engine/src/main/java/ai/grakn/engine/task/postprocessing/CountPostProcessor.java

```diff
@@ -120,13 +120,13 @@ public void updateCounts(CommitLog commitLog){
      *
      * @param keyspace The keyspace of the graph which the type comes from
      * @param conceptId The id of the concept with counts to update
-     * @param incrementBy The number of instances which the type has gained/lost
+     * @param value The number of instances which the type has gained/lost
      * @return true if sharding is needed.
      */
-    private static boolean incrementInstanceCountAndCheckIfShardingIsNeeded(RedisCountStorage redis, Keyspace keyspace, ConceptId conceptId, long incrementBy, long shardingThreshold){
-        long numShards = redis.getCount(RedisCountStorage.getKeyNumShards(keyspace, conceptId));
+    private static boolean incrementInstanceCountAndCheckIfShardingIsNeeded(CountStorage redis, Keyspace keyspace, ConceptId conceptId, long value, long shardingThreshold){
+        long numShards = redis.getShardCount(keyspace, conceptId);
```

Comment:
We're still calling this `redis` - I guess it should be `countStorage`.

Reply:
fixed!